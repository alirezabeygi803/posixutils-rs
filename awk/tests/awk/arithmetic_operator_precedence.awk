BEGIN {
    print 5 + 3 - 2;
    print 2 - 3 + 5;

    print 2 + 3 * 4;
    print 8 - 2 * 4;
    print 6 / 2 + 3;
    print 10 - 4 / 2;

    print 2 ^ 2 * 3;
    print 3 * 2 ^ 2;

    print (2 + 3) * 4;
    print 2 + (3 * 4);
    print (2 ^ 2) * 3;
    print 2 ^ (2 * 3);

    print 8 / 2 * 4;
    print 8 * 2 / 4;

    print -2 * 3;
    print -(2 * 3);
    print -2 ^ 2;
}