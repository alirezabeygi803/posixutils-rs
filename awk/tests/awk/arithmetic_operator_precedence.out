6
4
14
0
6
8
12
12
20
14
12
64
16
4
-6
-6
-4
